packages/data/addon/services/bard-dimensions.js

```diff
@@ -120,21 +118,19 @@ export default Service.extend({
       });
     }
 
-    return get(this, '_bardAdapter')
-      .all(dimension, options)
-      .then(recordsFromBard => {
-        let serialized = get(this, '_serializer').normalize(dimension, recordsFromBard),
-          dimensions = kegAdapter.pushMany(dimension, serialized, options);
+    return bardAdapter.all(dimension, options).then(recordsFromBard => {
+      let serialized = serializer.normalize(dimension, recordsFromBard),
+        dimensions = kegAdapter.pushMany(dimension, serialized, options);
 
-        // Fili provides pagination metadata only when data is partially fetched
-        let isPartiallyLoaded = get(recordsFromBard, 'meta.pagination');
-        if (!isPartiallyLoaded) {
-          this._setLoadedStatus(dimension);
-        }
+      // Fili provides pagination metadata only when data is partially fetched
+      let isPartiallyLoaded = get(recordsFromBard, 'meta.pagination');
```

Comment:
```suggestion
      const isPartiallyLoaded = get(recordsFromBard, 'meta.pagination');
```

Also do we need the getter here?

Reply:
fili does not always return a `meta` object so I believe it is required